ReactAndroid/src/main/java/com/facebook/react/jscexecutor/JSCExecutor.java

```diff
@@ -14,9 +14,18 @@
 import com.facebook.soloader.SoLoader;
 
 @DoNotStrip
-/* package */ class JSCExecutor extends JavaScriptExecutor {
+/* package */ public class JSCExecutor extends JavaScriptExecutor {
+  private static boolean loaded = false;
+
   static {
-    SoLoader.loadLibrary("jscexecutor");
+    loadLibrary();
+  }
+
+  public static void loadLibrary() throws UnsatisfiedLinkError {
+    if (!loaded) {
```

Comment:
Can you try removing this check and test as SoLoader.loadLIbrary has it's own check to prevent loading a library twice?

https://github.com/facebook/SoLoader/blob/main/java/com/facebook/soloader/SoLoader.java#L720

Reply:
Ah nice! The check removed.